frontend/packages/pipelines-plugin/src/components/pipelines/list-page/PipelineAugmentRunsWrapper.tsx

```diff
@@ -15,7 +15,10 @@ interface PipelineAugmentRunsWrapperProps {
 
 const PipelineAugmentRunsWrapper: React.FC<PipelineAugmentRunsWrapperProps> = (props) => {
   const pipelineData = _.get(props.pipeline, 'data', []);
-  if (pipelineData.length < 1) {
+  if (!props.pipeline.loaded) {
+    return null;
```

Comment:
As noted on the tests, this needs to be a loading spinner not `null`.

Reply:
Moved it to spinner.